setup.py

```diff
@@ -349,6 +353,7 @@ def do_setup():
             'azure_data_lake': azure_data_lake,
             'azure_cosmos': azure_cosmos,
             'azure_container_instances': azure_container_instances,
```

Comment:
I submit PR a few days ago. It was merged today. Look at: https://github.com/apache/airflow/pull/4524